CMakeLists.txt

```diff
@@ -23,7 +23,9 @@ FOR WINDOWS USE THIS GIT REPO https://github.com/git-for-windows/git
 #TODO Enable NLS on windows natively
 #TODO Add pcre support
 
-#NOTE
+#NOTE On Windows when compiling with MSVC or Clang you need add invalidcontinue.obj to the link options
```

Comment:
I think this note should be dropped from the code, and instead it should be explained in the commit message of the commit that introduces it.